Vitals/VitalStatusLogger.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.Diagnostics;
+
+namespace Vitals
+{
+    /// <summary>
+    /// This class prints a warning to the console 
+    /// incase of any breach in vitals
+    /// </summary>
+    public class VitalStatusLogger
+    {
+        public static void LogBreach(string vital,string level,float value)
+        {
+            Console.WriteLine(vital + " is " + level +"; "+ vital + " = " + value);
```

Comment:
Here, the LogBreach is logging to the console. Let's assume one client wants it this way, so you have implemented correctly.
What if another client wants to log over the network (so a central server can notify the concerned person)?
Will we need to make a copy of the entire program? Try to decouple the VitalStatusLogger from the VitalValueTester using an interface.